spec/unit/cookbook/syntax_check_spec.rb

```diff
@@ -159,12 +159,15 @@
       end
 
       describe "and a file has a syntax error" do
+
         before do
           cookbook_path = File.join(CHEF_SPEC_DATA, "cookbooks", "borken")
           syntax_check.cookbook_path.replace(cookbook_path)
         end
 
         it "it indicates that a ruby file has a syntax error" do
+          expect(Chef::Log).to receive(:fatal).with("Cookbook file default.rb has a ruby syntax error.")
```

Comment:
By using `basename` you're throwing away all the path information, I'd expect this to be `recipes/default.rb` ( and in some contexts it may need to be `<cookbook_name>/recipes/default.rb`, although I might be wrong about that).

Reply:
okay, I will look into it.  But in the case when we run the command from inside the cookbook directory like `cookbooks/first_cookbook` then we only get `recipes/default.rb` only if we run outside the cookbook directory we may get `<cookbook_name>/recipes/default.rb`.